envoy/stream_info/uint64_accessor.h

```diff
@@ -0,0 +1,26 @@
+#pragma once
+
+#include "envoy/common/pure.h"
+#include "envoy/stream_info/filter_state.h"
+
+namespace Envoy {
+namespace StreamInfo {
+
+/**
+ * A FilterState object that tracks a single uint64_t value.
+ */
+class UInt64Accessor : public FilterState::Object {
+public:
+  /**
+   * Increments the tracked value by 1.
+   */
+  virtual void increment() PURE;
```

Comment:
Why increment and not some other forms of integer mutation?

Reply:
This PR is basically a copy of UInt32Accessor (in the same folder), but changed 32 to 64. I guess that if needed we could add more APIs in the future? The main use case for this class is saving an integer value in a filter state

Reply:
Ah, fair.